drivers/platform/aducm3029/i2c.c

```diff
@@ -206,7 +214,22 @@ int32_t i2c_write(struct i2c_desc *desc,
 		return SUCCESS;
 	}
 
-	trans->bRepeatStart = (stop_bit == 1) ? 0 : 1;
+	if (stop_bit == 0) {
+		prologue_size = bytes_number;
+		if (prologue_data) {
+			temp_ptr = realloc(prologue_data, bytes_number);
```

Comment:
need to check realloc return

Reply:
Can a realloc(prologue_data, bytes_number + old_prologue_size) and then
memcpy(prologue_data + old_proloque_size, data, bytes_number) be a better approach?
This will enable write(stop_bit = 0) write(stop_bit = 0) and then read(stop_bit = 1).

Reply:
We can do that, but do we want to? What would be the reason to do this instead of just overwriting?